modules/serverless-workflow-traffic-splitting-flags.adoc

```diff
@@ -3,47 +3,47 @@
 //serverless/knative-client.adoc
 
 [id="traffice-splitting-flags_{context}"]
-= Traffic splitting flags 
+= Traffic splitting flags
 
 `kn` supports traffic operations on the traffic block of a service as part of the `kn service update` command.
 
-The following table displays a summary of traffic splitting flags, value formats, and the operation the flag performs. The "Repetition" column denotes whether repeating the particular value of flag is allowed in a `kn service update` command.
+The following table displays a summary of traffic splitting flags, value formats, and the operation the flag performs. The *Repetition* column denotes whether repeating the particular value of flag is allowed in a `kn service update` command.
 
 [cols="1,1,2,1",options="header"]
 |====
-| Flag 
-| Value(s) 
-| Operation 
+| Flag
+| Value(s)
+| Operation
 | Repetition
 
-|`--traffic` 
-| `RevisionName=Percent` 
-| Gives `Percent` traffic to `RevisionName` 
+|`--traffic`
+| `RevisionName=Percent`
+| Gives `Percent` traffic to `RevisionName`
 | Yes
 
-|`--traffic` 
-| `Tag=Percent` 
-| Gives `Percent` traffic to the Revision having `Tag` 
+|`--traffic`
+| `Tag=Percent`
+| Gives `Percent` traffic to the Revision having `Tag`
```

Comment:
Should the instances of un-marked-up "Revision" be lowercase throughout this table?

Reply:
Yes! Thanks for catching that